src/interventions/Interventions.jl

```diff
@@ -5,110 +5,100 @@ Base.@kwdef struct Intervention{N,P,P2} <: EcoModel
     seed_years::P2 = Param(
         10;
         ptype="integer",
         bounds=(5.0, 74.0 + 1.0, 5 / 70),
+        default_bounds=(5.0, 74.0 + 1.0, 5 / 70),
         dists="triang",
-        name="Years to Seed",
-        description="Number of years to seed for.",
-    )
+        name="Years to Seed", description="Number of years to seed for.")
     shade_years::P2 = Param(
         10;
         ptype="integer",
         bounds=(5.0, 74.0 + 1.0, 5 / 70),
+        default_bounds=(5.0, 74.0 + 1.0, 5 / 70),
         dists="triang",
-        name="Years to Shade",
-        description="Number of years to shade for.",
-    )
+        name="Years to Shade", description="Number of years to shade for.")
     plan_horizon::N = Param(
         5;
         ptype="integer",
         bounds=(0.0, 40.0 + 1.0),
+        default_bounds=(0.0, 40.0 + 1.0),
         dists="unif",
-        name="Planning Horizon",
-        description="How many years of projected data to take into account when selecting intervention locations (0 only accounts for current year).",
-    )
+        name="Planning Horizon", description="How many years of projected data to take into account when selecting intervention locations (0 only accounts for current year).")
     seed_freq::N = Param(
         5;
         ptype="integer",
         bounds=(0.0, 15.0 + 1.0),
+        default_bounds=(0.0, 15.0 + 1.0),
         dists="unif",
-        name="Seeding Frequency",
-        description="Frequency of seeding site selection (0 is set and forget).",
-    )
+        name="Seeding Frequency", description="Frequency of seeding site selection (0 is set and forget).")
     shade_freq::N = Param(
         1;
         ptype="integer",
         bounds=(0.0, 15.0 + 1.0),
+        default_bounds=(0.0, 15.0 + 1.0),
         dists="unif",
-        name="Shading Frequency",
-        description="Frequency of shading site selection (0 is set and forget).",
-    )
+        name="Shading Frequency", description="Frequency of shading site selection (0 is set and forget).")
     seed_year_start::N = Param(
         2;
         ptype="integer",
         bounds=(2.0, 25.0 + 1.0),
+        default_bounds=(2.0, 25.0 + 1.0),
         dists="unif",
-        name="Seeding Start Year",
-        description="Start seeding deployments after this number of years has elapsed.",
-    )
+        name="Seeding Start Year", description="Start seeding deployments after this number of years has elapsed.")
     shade_year_start::N = Param(
         2;
         ptype="integer",
         bounds=(2.0, 25.0 + 1.0),
+        default_bounds=(2.0, 25.0 + 1.0),
         dists="unif",
-        name="Shading Start Year",
-        description="Start of shading deployments after this number of years has elapsed.",
-    )
+        name="Shading Start Year", description="Start of shading deployments after this number of years has elapsed.")
```

Comment:
Minor issue but this is superfluous formatting - I preferred how it was before...

Reply:
This is how the formatter is formatting it, not sure how to revert it to how it was